app/views/branding_groups/list_navigate.html.erb

```diff
@@ -0,0 +1,44 @@
+<h2><%= link_to 'Home', site_path %> / Collections</h2>
```

Comment:
As an engineer, I appreciate that "/" can separate levels in a hierarchy, but in this less technical use case I interpreted it as "and / or".  We use a chevron elsewhere for this sort of thing, and it'd be less ambiguous here, too.